### PR TITLE
Update Do not caps-protect name fields #384.bibtex

### PR DESCRIPTION
**Pull request by nickbart1980**

### CODE REVIEW

File: test/fixtures/export/Do not caps-protect name fields #384.bibtex [hunk 1]
```diff
@@ -34,12 +34,12 @@
 }
 
 @book{bovendeert,
-  author = {{boven d'\relax{} Eert}, Christianus},
+  author = {{boven d'Eert}, Christianus},
```
Comment (retorquere) [inline test/fixtures/export/Do not caps-protect name fields #384.bibtex]: I don't understand... I thought you said NDPs need a `\relax ` and a space for BibTeX?

File: test/fixtures/export/Do not caps-protect name fields #384.bibtex [hunk 2]
```diff
@@ -179,22 +179,21 @@
...
 @book{este,
-  author = {d'\relax{} Este, Beatrice},
+  author = {d'\relax Este, Beatrice},
   note = {[Este][Beatrice d']}
 }
 
 @book{medici,
-  author = {de'\mbox{} Medici, Lorenzo},
-  note = {[Medici][Lorenzo de'\mbox]}
+  author = {de' Medici, Lorenzo},
```
Comment (retorquere) [inline test/fixtures/export/Do not caps-protect name fields #384.bibtex]: So is the `\mbox` only for BibLaTeX?

File: test/fixtures/export/Do not caps-protect name fields #384.bibtex (hunk 1 above)
Comment (nickbart1980) [inline test/fixtures/export/Do not caps-protect name fields #384.bibtex]: Only DPs do. Again, given the limitations of bibtex, the best that can be done (IMHO) is (1) to let bibtex parse DPs as prefixes (the space is needed for *parsing*, plus `\relax` to later remove this space betw. punctuation and Last) – and (2) let NDPs become part of a (multi-part) Last ({van Gogh}), but, optionally, sort by the original Last (Gogh).

File: test/fixtures/export/Do not caps-protect name fields #384.bibtex (hunk 2 above)
Comment (retorquere) [inline test/fixtures/export/Do not caps-protect name fields #384.bibtex]: Is it harmful or just unnecessary?

Comment (nickbart1980) [inline test/fixtures/export/Do not caps-protect name fields #384.bibtex]: for BibTeX: just unnecessary.

Comment (retorquere) [inline test/fixtures/export/Do not caps-protect name fields #384.bibtex]: Then I'd prefer to keep it in so I don't have to add too many exception-cases to the LaTeX generator.

Comment (retorquere) [inline test/fixtures/export/Do not caps-protect name fields #384.bibtex]: Can you add it back? I'm working towards the point where I can merge your test file.

Comment (nickbart1980) [inline test/fixtures/export/Do not caps-protect name fields #384.bibtex]: done

File: test/fixtures/export/Do not caps-protect name fields #384.bibtex [hunk 3]
```diff
@@ -179,22 +179,21 @@
 }
 
 @book{este,
-  author = {d'\relax{} Este, Beatrice},
+  author = {d'\relax Este, Beatrice},
```
Comment (nickbart1980) [inline test/fixtures/export/Do not caps-protect name fields #384.bibtex]: done

**Pull request merged**